### PR TITLE
TA-512 Task Portlet isn't displayed in FR

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: task-management/src/main/resources/locale/portlet/taskManagement_fr.properties [hunk 1]
```diff
@@ -56,7 +56,7 @@ label.yesterday=Hier
 label.today=Aujourd'hui
 label.tomorrow=Demain
 label.upcoming=Plus tard
-label.none=
+label.none=Aucun
```
Comment (thomasdelhomenie) [inline task-management/src/main/resources/locale/portlet/taskManagement_fr.properties]: The translations are managed by Crowdin directly. This change will be reverted during the next synchronization. We should fix the root cause of the bug instead.
Is this string still used ? Can't we remove it ?

Comment (boubaker) [inline task-management/src/main/resources/locale/portlet/taskManagement_fr.properties]: IMO the root cause is not a problem, in fact, it's caused by the fact that the empty string in I18N file are considered as null entries. We may have to consider this in org.exoplatform.task.util.TaskUtil.resolve(List<String>, ResourceBundle) and surround it by a try/catch block.

Comment (thomasdelhomenie) [inline task-management/src/main/resources/locale/portlet/taskManagement_fr.properties]: That's what I meant by "the root cause" :smile: No matter what the i18n file contains, it should not make the application fail.

Comment (boubaker) [inline task-management/src/main/resources/locale/portlet/taskManagement_fr.properties]: Done

File: services/src/main/java/org/exoplatform/task/util/TaskUtil.java [hunk 2]
```diff
@@ -134,7 +134,18 @@ private TaskUtil() {
       if (k.isEmpty()) {
         k = NONE;
       }
-      labels.put(k, bundle.getString("label." + k));
+      String label;
+      try {
+        label = bundle.getString("label." + k);
+      } catch (Exception e) {
+        if(LOG.isDebugEnabled()) {
+          LOG.warn("Error while resolving label '" + k + "' with locale " + bundle.getLocale(), e);
```
Comment (thomasdelhomenie) [inline services/src/main/java/org/exoplatform/task/util/TaskUtil.java]: We should keep it simple and also log a warn message with the stacktrace IMO.

Comment (boubaker) [inline services/src/main/java/org/exoplatform/task/util/TaskUtil.java]: Yep, as discussed I agree.

Comment (boubaker) [inline services/src/main/java/org/exoplatform/task/util/TaskUtil.java]: Done

**Comment by boubaker** [comment]
Merged

**Pull request closed without merging**